charts/karpenter/values.yaml

```diff
@@ -106,7 +106,6 @@ controller:
       cpu: 1
       memory: 1Gi
     limits:
-      cpu: 1
```

Comment:
Is this a good idea?

Reply:
I think it is. We recommend users not set cpu limits and Karpenter lends itself to small bursts in CPU on scale-ups. When I remove this and do large scale-ups, we can use several CPUs for a few seconds and then we drop down to next to nothing.  If we don't completely remove the limit, we definitely need to increase it.  On large scale-ups, CPU is throttled and we get weird timeouts.

Reply:
The challenge is that you have no guarantee that Karpenter is running on a node w/ more pods. Running Burstable pods and assuming they're Guaranteed is a key source of outages. T instance types are ideal for Karpenter's compute footprint.